### ISSUE TITLE
Dataset & training 

### ISSUE THREAD

**Issue by asia281** [Bunnybeibei/GexMolGen#2]
Hi,

I'm a master's student doing research on molecule generation and really would like to use your method as a baseline. I saw that in the todo there are two points: Upload the complete datase, Upload training code. Are you planning on doing that in the near future? Thanks in advance for the answer!

**Comment by Bunnybeibei** [comment]
Hi,

I apologize for the delayed response. We are currently not planning to make the training code public, as it is still being used for ongoing work. However, we will be releasing the complete test set soon.

Best regards.

**Comment by asia281** [comment]
Thanks. I'm looking forward to be able to use the full dataset then.

Kind regard
